docker/ubuntu-python3/build-and-install-scafacos.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+cd /tmp
+git clone --recursive git://github.com/scafacos/scafacos --branch dipoles
+cd scafacos
+./bootstrap
+./configure --enable-shared \
+	--with-internal-pfft --with-internal-pnfft \
+	--enable-fcs-solvers=direct,pnfft,p2nfft,p3m \
+	--disable-fcs-fortran --enable-fcs-dipoles
```

Comment:
Do you still need to install `gfortran` if fortran is  disabled?

Reply:
I've tried just now and apparently we need an MPI Fortran compiler nonetheless:
```
checking for mpic++... mpic++
checking whether we are using the GNU C++ compiler... yes
checking whether mpic++ accepts -g... yes
checking dependency style of mpic++... gcc3
checking for function MPI::Init... yes
checking for mpi.h... yes
checking for mpif95... no
checking for mpxlf95_r... no
checking for mpxlf95... no
checking for ftn... no
checking for mpif90... mpif90
checking whether we are using the GNU Fortran compiler... no
checking whether mpif90 accepts -g... no
checking for function MPI_INIT... no
checking for function MPI_INIT in -lmpichf90... no
checking for function MPI_INIT in -lfmpi... no
checking for function MPI_INIT in -lfmpich... no
configure: error: in `/tmp/scafacos':
configure: error: ScaFaCoS requires an MPI Fortran compiler.
See `config.log' for more details
The command '/bin/sh -c bash /tmp/build-and-install-scafacos.sh' returned a non-zero code: 1
```

Reply:
`--disable-fcs-fortran` means `Disable Fortran interface and tests`, but scafacos includes a couple Fortran libraries (`scafacos/lib/pepc`, `scafacos/lib/fmm`), which would explain the error message when `gfortran` is not installed.

Reply:
The wonders of autotools...